### DISCUSSION TITLE
uint16_t SX126x::getIrqStatus() is no longer available

### DISCUSSION THREAD

**Discussion by msfujino** [jgromes/RadioLib#1241] [Q&A] [answered]
**uint16_t SX126x::getIrqStatus()** which could be used in 6.6.0, is no longer available after 7.0.0 and later. Is there a reason for this?

**Comment by jgromes** [comment]
Due to some reworks to the way IRQs are handled in 7.0.0, this method has been changed to `uint32_t SX126x::getIrqFlags()` (to make the naming consistent across different modules), but it does the same thing that `uint16_t SX126x::getIrqStatus()` used to.

**Comment by msfujino** [comment, marked as answer]
I have confirmed that it can be compiled with 7.0.1. 
Thank you for your response.
